src/controllers/commentReaction.js

```diff
@@ -0,0 +1,46 @@
+import LikeComment from '../db/models';
+import errorResponse from '../helpers/index';
+
+const { CommentReaction } = LikeComment;
+
+/**
+ * @description this class handles methods for liking and unliking comment
+ */
+class ReactionController {
+  /**
+   * @description - This method is responsible for liking or unliking a comment
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} response - Response sent from the controller
+   * @returns {object} - object representing response message
+   * @memberof CommentReaction
+   */
+  static async likeOrUnlikeOneComment(request, response) {
+    try {
+      const { commentId } = request.params;
+      const userId = request.userData.payload.id;
+      const where = { $and: [{ commentId }, { likedBy: userId }] };
+
+      const commentReaction = await CommentReaction.findOne({ where });
+      if (commentReaction) {
+        await CommentReaction.destroy({ where });
```

Comment:
Nice job @Johnsonojo and @frostyblok . But I think destroying and recreating a comment reaction is not very effective. I suggest you update the `IsLiked` field instead. I will like to know what you think.

Reply:
I just thought about @okoroemeka suggestion. I think it's more effective not to delete the row after a user unlike a comment. The implication is that; every new like would create a row which would be assigned a unique id (even if the like and dislike is happening on the same comment, by the same user). Hence, if I liked and disliked a comment 50 times, I'd consume 50 IDs on your model.
Also, please change the `id` dataType to `BIGINT` so that it accommodates more likes, since we'd definitely have more likes than comments and articles in this project.

Reply:
Yeah @Johnsonojo that's the point we're actually driving. However, to make it clearer: A new row should be created when a user likes a comment, then the status of isLiked should change to false (rather than deleting the row) when the user unlikes the comment. If you delete the row, then a new row will always be created for the same user on that same comment, which is not efficient.
So, in essence, updating the row is better than deleting the row.